apps/src/music/views/Callouts.tsx

```diff
@@ -71,12 +71,11 @@ const availableCallouts: AvailableCallouts = {
   'trigger-button-1': {selector: `#${Triggers[0].id}`},
   'toolbox-first-row': {selector: '.blocklyTreeRow'},
   'flyout-first-block': {
-    selector:
-      '.blocklyFlyout:not([style*="display: none;"]) .blocklyDraggable:nth-of-type(1)',
+    selector: '.blocklyFlyout:not([style*="display: none;"]) .blocklyDraggable',
   },
   'toolbox-second-block': {
     selector:
-      '.blocklyFlyout:not([style*="display: none;"]) .blocklyDraggable:nth-of-type(3)',
+      '.blocklyFlyout:not([style*="display: none;"]) .blocklyDraggable ~ .blocklyDraggable',
```

Comment:
One would expect the second `.blocklyDraggable` to match on `:nth-of-type(2)`. Nothing matched on `:nth-of-type(1)` and unexpectedly the first `.blocklyDraggable` matched on `:nth-of-type(2)`.

The subsequent-sibling combinator (`~`) should match all occurrences of `.blocklyDraggable` that follow another `.blocklyDraggable`.  `document.querySelector` returns the first match, which should be the second `.blocklyDraggable`.